frontend_tests/node_tests/unread.js

```diff
@@ -191,15 +191,15 @@ var zero_counts = {
 
     var num_msgs = 10000;
     var i;
-    for (i = 0; i < num_msgs; ++i) {
+    for (i = 0; i < num_msgs; i+=1) {
```

Comment:
Do I have to put spaces around these too? Like: `for (i = 0; i < num_msgs; i += 1)`?